### PR TITLE
feat: api

### PR DESCRIPTION
**Pull request by 0xArdy**

### CODE REVIEW

File: src/hooks/useTransactions/index.ts [hunk 1]
```diff
@@ -109,6 +115,7 @@ export const useTransactions = () => {
       const error = e as BaseError;
 
       setTxMetadata({ step: TransactionStep.NONE });
+      setErrorTitle('Transaction failed');
```
Comment (0xtiti) [inline src/hooks/useTransactions/index.ts:118]: I think locale is missing in errorTitle

Comment (0xtiti) [reply src/hooks/useTransactions/index.ts:118]: Also, probably should be added on dependency array to reuse in other components

**Review by 0xtiti — DISMISSED** [review]
lgtm! Just wondering if feat: api is the best naming for the PR 🤔

File: src/hooks/useTransactions/index.ts (hunk 1 above)
Comment (0xArdy) [reply src/hooks/useTransactions/index.ts]: https://github.com/defi-wonderland/superchain-bridge-ui/pull/88/commits/9c6c51ecfc03eff820edefa9fad08d0472c2d8f4

**Review by 0xtiti — DISMISSED** [review]

File: src/components/Connect.tsx [hunk 2]
```diff
@@ -21,17 +28,52 @@ export const Connect = ({ ...props }: ButtonProps) => {
     }
   };
 
+  const getData = useCallback(async () => {
+    if (!address) return;
```
Comment (turtlemoji) [inline src/components/Connect.tsx:32]: redundant check (you already check in parent function that calls it

**Review by turtlemoji — APPROVED** [review]

**Review by 0xtiti — APPROVED** [review]

**Pull request merged**